R/use_standalone.R

```diff
@@ -53,6 +53,9 @@
 #' }
 use_standalone <- function(repo_spec, file = NULL, ref = NULL, host = NULL) {
   check_is_project()
+  check_string(file, allow_null = TRUE)
+  check_string(ref, allow_null = TRUE)
+  check_string(host, allow_null = TRUE)
```

Comment:
I think the checks for `file` and `host` could be `maybe_name()`, allowing `NULL` but not empty or `NA`? I think `ref` can allow `""`

Reply:
I've used `maybe_name()` for all three to keep things simple.